src/common/header/components/GitHubButton.tsx

```diff
@@ -0,0 +1,21 @@
+import { IconButton } from '@chakra-ui/button';
+import { AiFillGithub } from 'react-icons/ai';
+
+export function GitHubButton(): JSX.Element {
+  return (
+    <IconButton
+      as="a"
+      href="https://github.com/VikeLabs/courseup"
+      target="_blank"
+      size="sm"
+      colorScheme="none"
+      fontSize="35px"
+      isRound
+      aria-label="Open GitHub Repo"
+      _hover={{
+        color: '#D3D3D3',
```

Comment:
Chakra tip: A lot of the time you don't have to hard code colour codes in because chakra offers a lot of predefined, https://chakra-ui.com/docs/theming/theme#gray
So you could say `gray.500` or whatever works here instead of `#D3D3D3`

Reply:
smh... can you tell I'm new to all this? hahah I've [committed a change](https://github.com/VikeLabs/courseup/pull/257/commits/f6544c273c802249c5d183d5c0170c6b053919a4) for that! Let me know if there's anything else and sorry about all these silly mistakes

Reply:
No need to apologize, learning is all part of it!